IFC4x3/Sections/Domain specific data schemas/Schemas/IfcPortsAndWaterwaysDomain/PropertySets/Pset_GateHeadCommon/DocPropertySet.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_GateHeadCommon" UniqueId="8dbb748d-b6f3-4559-8587-0575f097df30" ApplicableType="IfcFacilityPart/IfcMarinePartTypeEnum(.GATEHEAD.)" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_GateHeadCommon" UniqueId="8dbb748d-b6f3-4559-8587-0575f097df30" ApplicableType="IfcFacilityPart/IfcMarinePartTypeEnum(.GATEHEAD.)" PropertySetType="PSET_OCCURRENCEDRIVEN">
```

Comment:
This ApplicableType-string would probably change to "IfcMarineFacilityPart/GATEHEAD" after the redesign of facility part structure.

Reply:
Yes, indeed. But we are handling psets in a separate PR, so let us leave it.